IHP/IDE/CodeGen/ApplicationGenerator.hs

```diff
@@ -133,22 +131,30 @@ generateGenericApplication applicationName =
                 <> "</body>\n"
                 <> "|]\n"
                 <> "\n"
+                <> "stylesheets = do\n"
+                <> "    when (isDevelopment FrameworkConfig.environment) [hsx|\n"
+                <> "      <link rel=\"stylesheet\" href=\"/vendor/bootstrap.min.css\"/>\n"
+                <> "      <link rel=\"stylesheet\" href=\"/vendor/flatpickr.min.css\"/>\n"
+                <> "      <link rel=\"stylesheet\" href=\"/app.css\"/>\n"
+                <> "    |]\n"
+                <> "    when (isProduction FrameworkConfig.environment) [hsx|\n"
+                <> "      <link rel=\"stylesheet\" href=\"/prod.css\"/>\n"
+                <> "    |]\n"
+                <> "\n"
                 <> "scripts = do\n"
-                <> "    when (isDevelopment FrameworkConfig.environment) [hsx|<script id=\"livereload-script\" src=\"/livereload.js\"></script>|]\n"
-                <> "    [hsx|\n"
-                <> "        <script src=\"/vendor/morphdom-umd.min.js\"></script>\n"
-                <> "        <script src=\"/vendor/jquery-3.2.1.slim.min.js\"></script>\n"
-                <> "        <script src=\"/vendor/timeago.js\"></script>\n"
-                <> "        <script src=\"/vendor/popper.min.js\"></script>\n"
-                <> "        <script src=\"/vendor/bootstrap.min.js\"></script>\n"
-                <> "        <script src=\"/helpers.js\"></script>\n"
+                <> "    when (isDevelopment FrameworkConfig.environment) [hsx|\n"
+                <> "      <script id=\"livereload-script\" src=\"/livereload.js\"></script>\n"
+                <> "      <script src=\"/vendor/jquery-3.2.1.slim.min.js\"></script>\n"
+                <> "      <script src=\"/vendor/timeago.js\"></script>\n"
+                <> "      <script src=\"/vendor/popper.min.js\"></script>\n"
+                <> "      <script src=\"/vendor/bootstrap.min.js\"></script>\n"
+                <> "      <script src=\"/vendor/flatpickr.js\"></script>\n"
+                <> "      <script src=\"/helpers.js\"></script>\n"
+                <> "      <script src=\"/vendor/morphdom-umd.min.js\"></script>\n"
                 <> "    |]\n"
                 <> "    when (isProduction FrameworkConfig.environment) [hsx|\n"
-                <> "            <script src=\"/vendor/turbolinks.js\"></script>\n"
-                <> "            <script src=\"/vendor/morphdom-umd.min.js\"></script>\n"
-                <> "            <script src=\"/vendor/turbolinksMorphdom.js\"></script>\n"
-                <> "            <script src=\"/vendor/turbolinksInstantClick.js\"></script>\n"
-                <> "        |]\n"
+                <> "       <script src=\"/prod.js\"></script>\n"
```

Comment:
and here :)

Reply:
Shocker. Sorry, this PR has been a mess.